.php_cs.dist

```diff
@@ -9,11 +18,17 @@ return PhpCsFixer\Config::create()
         '@Symfony' => true,
         '@Symfony:risky' => true,
         'array_syntax' => array('syntax' => 'long'),
+        'no_unreachable_default_argument_value' => false,
+        'braces' => array('allow_single_line_closure' => true),
+        'heredoc_to_nowdoc' => false,
+        'phpdoc_annotation_without_dot' => false,
+        'protected_to_private' => false,
```

Comment:
we should add also this rule:
+        'no_break_comment' => true,

Reply:
I meant "false"

Reply:
:+1: it is however a PSR2 rule (the strangest IMHO but still) ;)
I'll remove it here